databricks/koalas/series.py

```diff
@@ -28,6 +28,7 @@
 from pyspark.sql.types import FloatType, DoubleType, LongType, StructType, TimestampType, \
     to_arrow_type
 
+from databricks import koalas as ks
```

Comment:
Is this required?

Reply:
It's required for PyCharm IDE to run doctests (but not the library itself)

1. PyCharm's doctest running requires it.
2. PyCharm reference resolving requires it

(see before after images in PR description)

Reply:
It is not required if we run the tests via terminal.

Reply:
OK I'd then leave a comment here, because otherwise people might remove it in the future.